File: src/Person.java
public class Person {

        private String name;
        private Department department;
        
        public Person(String name, Department department) {
            this.name = name;
            this.department = department;
        }

        public String setName() {
            this.name = name;
        }

        public String getName() {
            return name;
        }

        public Department setDepartment() {
            this.department = department;
        }

        public Department getDepartment() {
            return department;
        }

        public String getFaculty() {
            return department.getFaculty();
        }
    }

File: Iter3/src/Person.py
import department


class Person:
    __name: str
    __department: department

    def __init__(self, name, department):
        self.__name = name
        self.__department = department

    def get_name(self):
        return self.__name

    def get_department(self):
        return self.__department

    def get_faculty(self):
        return self.__department.get_faculty()
